docs/pages/admin-guides/management/guides/migrating-iam-ic-from-okta-to-teleport.mdx

```diff
@@ -0,0 +1,211 @@
+
+![Migration Mid Point](../../../../img/identity-center/ic-migrate-mid.png)
+
+### Ending point
+
+Once Teleport is in control of all the users and groups you want to provision
+into Identity Center, the Okta Provisioning and Push Groups functions can be
+retired, leaving Okta only providing SSO login.
+
+![Migration End Point](../../../../img/identity-center/ic-migrate-end.png)
+
+## Step 1: Install Okta SAML connector
+
+Install Okta SAML connector into Teleport as per the Teleport 
+[Okta as an SSO provider](../../access-controls/sso/okta.mdx) guide.
+
+<Admonition type="note">
+ Ideally, both Identity Center and Teleport will use the same Okta SAML application
+ for their signin, which guarantees that both Okta and Teleport will view the same 
+ user set. 
+
+ If this is not possible, the most flexible approach we've found is having an
+ an Okta Group for those users that should have IC access, and then assigning
+ both the Okta Identity Center App and the Okta SAML App used for Teleport SAML
+ connector to that group. This ensures that the same user set is visible across
+ both applications.
+</Admonition>
+
+You will need the Teleport SAML Connector Name and Okta SAML App ID in the next
+step.
+
+## Step 2: Install the Teleport Okta integration
+
+We will be using a very limited subset of the Teleport Okta integration in this 
+deployment, disabling all features except periodic user synchronization. This
+configuration is not supported by the normal installation UI, so we will have to
+use `tctl` to install it:
+
+```console
+$ ./tctl plugins install okta \
+    --org ${OKTA_ORG_URL} \
+    --saml-connector ${TELEPORT_SAML_CONNECTOR_NAME} \
+    --app-id ${OKTA_SAML_APP_ID} \
+    --api-token ${OKTA_API_TOKEN} \
+    --no-scim \
+    --no-accesslist-sync \
+    --no-appgroup-sync
+```
+
+This will install the Okta integration and start the User sync service. You can
+monitor the state of the Okta integration in the Teleport Integrations UI.
+
+## Step 3: Wait for user sync
+
+To make sure everything is working, wait until the first Okta to Teleport user
+sync has occurred. You can verify this by either
+ - refreshing the user page and finding your Okta users,
+ - checking the Okta integration status page, or
+ - watching the Teleport logs
+
+Once your Okta users are imported into Teleport, you can progress to the next
+step:
+
+## Step 4: Install the Teleport Identity Center Integration.
+
+Again, we need to install the plugin using `tctl`. 
+
+```console
+$ ./tctl plugins install awsic \
+    --arn ${IDENTITY_CENTER_INSTANCE_ARN} \
+    --region ${IDENTITY_CENTER_INSTANCE_REGION} \
+    --use-system-credentials \
+    --url ${IDENTITY_CENTER_SCIM_BASE_URL} \
+    --token ${IDENTITY_CENTER_SCIM_BEARER_TOKEN} \
+    --default-owner ${TELEPORT_ACCESS_LIST_DEFAULT_OWNER} \
+    --user-origin okta \
+    --account-name ${ACCOUNT_NAME_ALLOW_FILTER} \
+    --group-name ${GROUP_NAME_ALLOW_FILTER}
```

Comment:
A logical question that arises, how can these filters be updated after the integration has been created? I didn't see a mention of this later in the guide.